Makefile

```diff
@@ -346,29 +346,41 @@ build-manifest: kustomize
 	./hack/build-manifest.sh "${OPTS}"
 .PHONY: build-manifest
 
-cluster-clean: build-manifest
-	./hack/cluster-clean.sh
-.PHONY: cluster-clean
```

Comment:
Why the cluster clean was removed? This is for development purpose to delete Kepler.

Reply:
To delete everything on the cluster I have added [make cluster-restart](https://github.com/sustainable-computing-io/kepler/pull/909/files#:~:text=cluster%2Drestart-,cluster,-%2Drestart%3A) which will delete everything locally and re-run the cluster as fresh. After this user can run make cluster-deploy which will build/push Kepler to a local registry. and deploy Kepler using the image that is built. The cluster-clean functionality still remains.